content/source/docs/enterprise/run/install-software.html.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: "enterprise2"
+page_title: "Installing Software - Runs - Terraform Enterprise"
+sidebar_current: "docs-enterprise2-run-install"
+---
+
+# Installing Custom Software
+
+In some cases, it may be useful to install certain software on the Terraform worker, 
+such as a configuration management tool or cloud CLI.
+
+Terraform's `local-exec` [provisioner](https://www.terraform.io/docs/provisioners/local-exec.html) can be used in Terraform Enterprise (TFE). This will execute commands on the host machine running Terraform. In general, it should be used in conjunction with the resource(s) the software will be used to interact with, as relying on a placeholder (such as a null resource) may result in inconsistencies as to whether the software is installed.
+
+```hcl
+resource "aws_instance" "example" {
+  ami           = "${var.ami}"
+  instance_type = "t2.micro"
+  provisioner "local-exec" {
+    command = <<EOH
+sudo apt-get update
+sudo apt-get install -y ansible
+EOH
+  }
+}
+```
+
+An [explicit `depends_on`](/intro/getting-started/dependencies.html#implicit-and-explicit-dependencies) may sometimes be needed to place the installation correctly in the provisioning sequence.
```

Comment:
This should be removed, as we are no longer talking about making a resource depend on a `null_resource`.